pkgs/by-name/tu/tuxguitar/package.nix

```
@@ -34,9 +39,12 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s $out/dist $out/bin/dist
     ln -s $out/lib $out/bin/lib
     ln -s $out/share $out/bin/share
+  '';
 
+  postFixup = ''
     wrapProgram $out/bin/tuxguitar \
-      --set JAVA "${jre}/bin/java" \
+      "''${gappsWrapperArgs[@]}" \
+      --prefix PATH : ${lib.makeBinPath[ jre which ]} \
```

```suggestion
      --prefix PATH : ${lib.makeBinPath [ jre which ]} \
```
To make `nixfmt` happy